media-api/app/controllers/MediaApi.scala

```diff
@@ -348,7 +349,8 @@ object SearchParams {
       request.getQueryString("free").map(_.toBoolean),
       request.getQueryString("uploadedBy"),
       commaSep("labels"),
-      commaSep("hasMetadata")
+      commaSep("hasMetadata"),
+      request.getQueryString("newCostModel").nonEmpty
```

Comment:
I'm using it as a flag, as it's off by default (i.e. doesn't have a `false` option). So the QS would be `/search?newCostModel`. Don't really care as it's temporary, but felt like a flag, not an option.